nixos/modules/services/security/fwknopd.nix

```diff
@@ -0,0 +1,268 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.services.fwknopd;
+
+  ## Transform an attrset to a fwknopd config file.
+  toFwknopd = kv:
+    let renderKeyValue = k: v:
+          if v != null && v != [] then k + " " + renderValue v + "\n"
+          else "";
+        renderValue = v:
+          let err = t: v: abort
+            ("toFwknopd: " +
+             "${t} not supported: ${lib.toPretty {} v}");
+          in   if lib.isInt      v then toString v
```

Comment:
Beautification is ugly.

Reply:
@AndersonTorres, it's way easier to scan visually this way -- this is essentially a table relationship.

Is this is a show-stopper for you?

Reply:
To be honest, yes. But there are more serious things to worry.